metrics/lib/common.bash

```diff
@@ -92,6 +92,20 @@ check_images()
 	done
 }
 
+# This function performs a docker build on the image names
+# passed in, to ensure that we have the latest changes from
+# the dockerfiles
+check_dockerfiles_images()
+{
+	local image="$1"
+	local dockerfile_path="$2"
+	local exists=$(docker image inspect $image > /dev/null; echo $?)
```

Comment:
Is this function called with `errexit` flag (that probably would be good to do)?. If yes the echo may not be executed?